pkg/apis/componentconfig/cloudcore/v1alpha1/types.go

```diff
@@ -408,6 +410,35 @@ type NodeUpgradeJobControllerLoad struct {
 	NodeUpgradeJobWorkers int32 `json:"nodeUpgradeJobWorkers,omitempty"`
 }
 
+// ImagePrepullController indicates the operations controller
+type ImagePrepullController struct {
```

Comment:
```suggestion
type ImagePrePullController struct {
```
`ImagePrePull` should be better?

Reply:
Is it better to change the entire API definition to PrePull?

Reply:
> Is it better to change the entire API definition to PrePull?

all `prepull` have been change to `PrePull`